tests/foreman/ui/test_subscription.py

```diff
@@ -190,6 +190,52 @@ def test_positive_access_with_non_admin_user_with_manifest(test_name):
                 == DEFAULT_SUBSCRIPTION_NAME)
 
 
+@tier2
+def test_positive_access_manifest_as_another_admin_user(test_name):
+    """Other admin users should be able to access and manage a manifest
+    uploaded by a different admin.
+
+    :id: 02e319da-3b7a-4694-9164-475c2c71b9a8
+
+    :expectedresults: Other admin user should see/manage the manifest
+
+    :BZ: 1669241
+
+    :customerscenario: true
+
+    :CaseLevel: Integration
+
+    :CaseImportance: High
+    """
+    org = entities.Organization().create()
+    user1_password = gen_string('alphanumeric')
+    user1 = entities.User(
+        admin=True,
+        password=user1_password,
+        organization=[org],
+        default_organization=org,
+    ).create()
+    user2_password = gen_string('alphanumeric')
+    user2 = entities.User(
+        admin=True,
+        password=user2_password,
+        organization=[org],
+        default_organization=org,
+    ).create()
+    # use the first admin to upload a manifest
+    with Session(test_name, user=user1.login, password=user1_password) as session:
+        manifests.upload_manifest_locked(org.id)
```

Comment:
@rochacbruno , while doing a UI + CLI part of this, I noticed that this action isn't actually ran as user specified in the Session - for a simple reason, Session is an Airgun thing and this call is not an Airgun call. This action is actually ran as a user specified in robottelo.properties.

Reply:
@lhellebr good catch!

we have 3 options

1. leave as it is if there is no problem uploadin as admin

2. accept user argument

3. infere user argument by session context.

@JacobCallahan ?

Reply:
Option 4: upload manifest using UI. See `tests/foreman/ui/test_subscription.py::test_select_customizable_columns_uncheck_and_checks_all_checkboxes` for example how it can be done.

Option 1 means that we create user that we don't need. Wasted second or two, and unnecessary code in test.